06 piškvorky/piskvorky1D.py

```diff
@@ -0,0 +1,77 @@
+from random import randrange
+import time
+
+# TICK-TACK-TOE
+
+
+def score(area):  # fce get list(area) and return result in accordance with area
+
+    # draw
+    if '-' not in area:
+        print(' "!" ')
+        quit()
+    # players won
+    elif (3 * player_symbol) in area:
+        print(' "X" ')
+        quit()
+    # pc won
+    elif (3 * pc_symbol) in area:
+        print('"O"')
+        quit()
+    # next play
+    else:
+        print('"-"')
+
+
+def move(area, position, symbol):  # put symbol into area, get area, position 1 - 19, symbol
+
+    area = list(area)
+    area[position - 1] = symbol
+    area = ''.join(area)
+    return area
+
+
+def player_move(area):  # get input(position) from player, test input and return area with symbol
+
+    position = input('Kam umístit X? Vyber políčko 1 - 20. Tvá volba: ')
+    while True:
+
+        position = str(position)
+        if position.isalpha():
+            position = int(input("Políčko vybírej jen pomocí celých, kladných čísel. Políčko: "))
+        position = int(position)
+
+        while position <= 0 or position >= (len(area)+1):
```

Comment:
Ja by som zvážila pre zjednodušenie v kóde upravovať o 1tku užívateľov výber a všetko ostatné by si mohla nechať bez posúvania (ťah počítača, veľkosť hracieho poľa). T.j. ak ti užívateľ zadá 5ku, pracuj s ňou ako so 4kou :).

Reply:
Ten while tu je zbytočný a skôr ti škodí. Ak totiž užívateľ zadá číslo ktoré bude veľké a zacyklí sa do tohoto while, pri druhom pokuse a zadaní napr. písmenka, ti to spadne pretože už nebeží validácia vstupu.